watertap/property_models/seawater_prop_pack.py

```diff
@@ -323,93 +323,142 @@ def build(self):
+            units=enth_mass_units * P_inv_units * t_inv_units,
             doc="Specific enthalpy parameter A2",
         )
         self.enth_mass_param_A3 = Var(
             within=Reals,
-            initialize=-0.535,
-            units=enth_mass_units * t_inv_units**2,
+            initialize=0.0127,
+            units=enth_mass_units * P_inv_units * t_inv_units**2,
             doc="Specific enthalpy parameter A3",
         )
         self.enth_mass_param_A4 = Var(
             within=Reals,
-            initialize=0.004,
-            units=enth_mass_units * t_inv_units**3,
+            initialize=-4.7723e-5,
+            units=enth_mass_units * P_inv_units * t_inv_units**3,
             doc="Specific enthalpy parameter A4",
         )
+        self.enth_mass_param_A5 = Var(
+            within=Reals,
+            initialize=-1.1748,
+            units=enth_mass_units * P_inv_units,
+            doc="Specific enthalpy parameter A5",
+        )
+        self.enth_mass_param_A6 = Var(
+            within=Reals,
+            initialize=0.01169,
+            units=enth_mass_units * P_inv_units * t_inv_units,
+            doc="Specific enthalpy parameter A6",
+        )
+        self.enth_mass_param_A7 = Var(
+            within=Reals,
+            initialize=-2.6185e-5,
+            units=enth_mass_units * P_inv_units * t_inv_units**2,
+            doc="Specific enthalpy parameter A7",
+        )
+        self.enth_mass_param_A8 = Var(
+            within=Reals,
+            initialize=7.0661e-8,
+            units=enth_mass_units * P_inv_units * t_inv_units**3,
+            doc="Specific enthalpy parameter A8",
+        )
         self.enth_mass_param_B1 = Var(
             within=Reals,
-            initialize=-2.348e4,
+            initialize=-2.34825e4,
             units=enth_mass_units,
             doc="Specific enthalpy parameter B1",
         )
         self.enth_mass_param_B2 = Var(
             within=Reals,
-            initialize=3.152e5,
+            initialize=3.15183e5,
             units=enth_mass_units,
             doc="Specific enthalpy parameter B2",
         )
         self.enth_mass_param_B3 = Var(
             within=Reals,
-            initialize=2.803e6,
+            initialize=2.80269e6,
             units=enth_mass_units,
             doc="Specific enthalpy parameter B3",
         )
         self.enth_mass_param_B4 = Var(
             within=Reals,
-            initialize=-1.446e7,
+            initialize=-1.44606e7,
             units=enth_mass_units,
             doc="Specific enthalpy parameter B4",
         )
         self.enth_mass_param_B5 = Var(
             within=Reals,
-            initialize=7.826e3,
+            initialize=7.82607e3,
             units=enth_mass_units * t_inv_units,
             doc="Specific enthalpy parameter B5",
         )
         self.enth_mass_param_B6 = Var(
             within=Reals,
-            initialize=-4.417e1,
+            initialize=-4.41733,
```

Comment:
Whoa, so the previous parameter was off by an order of magnitude? It used to be -4.417e1 and now its -4.41733.

Reply:
I haven't checked the literature source for the older values to see if they were input incorrectly, but the values I have here are correct according to the new literature source.
![image](https://user-images.githubusercontent.com/96305519/210593647-a1041ffa-4c52-45c4-b1ef-6cf2f177c47d.png)

Reply:
These are the parameter values for the old source, so it looks like they were input correctly and these two values just differ by an order of magnitude for some reason.
![image](https://user-images.githubusercontent.com/96305519/210596196-ad6a7cd5-609f-44e8-99ce-3a321f1a3b40.png)